gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcMembershipRepository.java

```diff
@@ -98,6 +98,21 @@ public List<String> deleteByReferenceIdAndReferenceType(String referenceId, Memb
         }
     }
 
+    @Override
+    public List<Membership> findByReferenceIdAndReferenceType(String referenceId, MembershipReferenceType referenceType)
+        throws TechnicalException {
+        LOGGER.debug("JdbcMembershipRepository.deleteByReferenceIdAndReferenceType({}, {})", referenceId, referenceType);
```

Comment:
Could you check it seems the message is not updated it is still mentioning delete instead of find. But maybe there is something wrong with the GitHub UI